docs/user-guide/connectors/twilio.rst

```diff
@@ -39,7 +39,10 @@ Connecting to WhatsApp
 You can deploy a Rasa Open Source assistant to WhatsApp through Twilio. However, to do so, you have
 to have a `WhatsApp Business <https://www.whatsapp.com/business/>`_ profile. Associate
 your Whatsapp Business profile with the phone number you purchased through Twilio to
-access the `Twilio API for WhatsApp <https://www.twilio.com/whatsapp>`_.
+access the `Twilio API for WhatsApp <https://www.twilio.com/docs/whatsapp/api>`_.
+
+Note that according to the `API documentation <https://www.twilio.com/docs/whatsapp/api#using-phone-numbers-with-whatsapp>`_, 
+the phone number you use should be prefixed with `whatsapp+` in the ``credentials.yml`` described below.
```

Comment:
```suggestion
the phone number you use should be prefixed with `whatsapp:` in the ``credentials.yml`` described below.
```

Reply:
smh i thought i ran into this issue and fixed it myself but apparently only in the yaml file, thank you